apisix/plugins/openid-connect.lua

```diff
@@ -101,6 +101,7 @@ local schema = {
 local _M = {
     version = 0.1,
     priority = 2599,
+    type = 'auth',
```

Comment:
🤔 This is not belonged to Auth plugin AFAIK, there

Reply:
Oh yes yes, I forgot